src/redux/books/books.js

```diff
@@ -0,0 +1,37 @@
+const ADD_BOOK = 'ADD_BOOK';
+const DELETE_BOOK = 'DELETE_BOOK';
```

Comment:
- [ ] Good job so far 👍 Kindly make sure that the values of your action follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). Therefore it should be something like this: const `ADD_BOOK = bookstore/books/ADD_BOOK`. Kindly rectify this 💯